signal/src/fft_auto_scale.cc

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "signal/src/fft_auto_scale.h"
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "signal/src/max_abs.h"
+#include "signal/src/msb.h"
+
+// TODO(b/286250473): remove namespace once de-duped libraries
+namespace tflm_signal {
+
+int FftAutoScale(const int16_t *input, int size, int16_t *output) {
+  const int16_t max = tflm_signal::MaxAbs16(input, size);
```

Comment:
The `tflm_signal::` prefix shouldn't be necessary here or line 29, as we're already in that namespace.

Reply:
Done.